platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/engine/CryptoEngine.java

```diff
@@ -214,7 +214,8 @@ public Hash digestSync(final SelfSerializable serializable, final DigestType dig
     public Hash digestSync(
             final SerializableHashable serializableHashable, final DigestType digestType, final boolean setHash) {
         try {
-            final Hash hash = serializationDigestProvider.compute(serializableHashable, digestType);
+            final Hash hash =
+                    new Hash(serializationDigestProvider.compute(serializableHashable, digestType), digestType);
```

Comment:
can you split this in 2 lines by storing `serializationDigestProvider.compute(serializableHashable, digestType)` in a var?

Reply:
done